circleci/docker-publish

```diff
@@ -15,7 +15,6 @@ check_hub_vars() {
   # User supplied args
   if [[ -z $DOCKER_USER ]]; then echo "Missing arg1 DOCKER_USER" && exit 1; fi
   if [[ -z $DOCKER_PASS ]]; then echo "Missing arg2 DOCKER_PASS" && exit 1; fi
-  if [[ -z $DOCKER_EMAIL ]]; then echo "Missing arg3 DOCKER_EMAIL" && exit 1; fi
   if [[ -z $ORG ]]; then echo "Missing arg4 ORG" && exit 1; fi
```

Comment:
is now arg3? or do we need to handle either case?

Reply:
Ah yeah, I think we should still keep this validation logic, even though we don't use it